arrow-libs/resilience/arrow-resilience/src/commonMain/kotlin/arrow/resilience/Schedule.kt

```diff
@@ -33,9 +30,6 @@ import kotlin.time.Duration
 import kotlin.time.Duration.Companion.ZERO
 import kotlin.time.Duration.Companion.nanoseconds
 
-public typealias ScheduleStep<Input, Output> =
```

Comment:
Nit: This technically breaks anyone who used `ScheduleStep` in their code (and similarly anyone who called `val f = Schedule.step`) but I don't think it's worth the worry

Reply:
I've managed to rewrite it so that `ScheduleStep` and `step` are still there :)